features/checkout/paying_for_order/selecting_order_payment_method.feature

```diff
@@ -0,0 +1,21 @@
+@checkout
+Feature: Selecting an order payment method
+    In order to pay for my order
+    As a Customer
+    I want to be able to choose a payment method
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying with "Offline"
+        And I am logged in as customer
+
+    @ui
+    Scenario: Selecting a payment method
+        Given I have product "PHP T-Shirt" in the cart
+        And I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I select "Free" shipping method
+        And I am at the checkout payment step
```

Comment:
This step simply opens payment step page. To pass whole checkout properly, we should use use step:
```gherkin
And I complete the shipping step
```